### PR TITLE
feature: S3C-2790 get object lock configuration

### PR DESCRIPTION
**Pull request by ilkescality**
This PR includes GET Object Lock Configuration API and unit tests.

Related to https://github.com/scality/Arsenal/pull/1009 which includes the route, custom error, and a helper class method.

### CODE REVIEW

File: lib/api/bucketGetObjectLock.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                method: 'bucketGetObjectLock',
+            });
+            return callback(err, null, corsHeaders);
+        }
+        const objectLockEnabled = bucket._objectLockEnabled;
+        const objectLockConfig = bucket._objectLockConfiguration ?
+            bucket._objectLockConfiguration :
+            {};
+        if (!objectLockEnabled) {
+            log.debug('error processing request', {
```
Comment (jonathan-gramain) [inline lib/api/bucketGetObjectLock.js]: Even if it's a debug log, could have a more explanatory message saying that the object lock is not enabled.

**Review by jonathan-gramain — APPROVED** [review]

**Review by dora-korpar — APPROVED** [review]

**Comment by ilkescality** [comment]
@bert-e reset

**Comment by ilkescality** [comment]
@bert-e reset

**Pull request merged**